ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |-
+    FROM quay.io/fuse_qe/syndesisqe-tests:1.14.x
```

Comment:
Do we still need to be testing this older version?

Reply:
This is touching on the same topic being discussed here
https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1689177285378299

Reply:
@ascerra Was originally developed and tested here.  Just yesterday was asked to move to 1.15.  Want to keep for historical purposes n -1. I was expecting to rm when 1.16 comes out.  Also, was unsure if they wanted to run both. Asked Badre and he indicated to only run the latest.

Reply:
removed